docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.field.node.cgov_mini_landing.field_landing_contents.yml

```diff
@@ -36,10 +37,11 @@ settings:
       cgov_card_borderless: cgov_card_borderless
       cgov_card_raw_html: cgov_card_raw_html
       cgov_dynamic_list: cgov_dynamic_list
-      ncids_3_feature_row_opt_head: ncids_3_feature_row_opt_head
-      ncids_content_block: ncids_content_block
       ncids_page_title_block: ncids_page_title_block
       ncids_slim_hero: ncids_slim_hero
+      ncids_3_feature_row_opt_head: ncids_3_feature_row_opt_head
```

Comment:
Can you please run `drush features:list:packages` and attach to this PR? This ordering is off and features would usually mark this as having issues.

Reply:
Just to follow up on this, @dev-rana-publicis  showed me his instance, and features had no issues with this file.